src/main/java/com/google/step/youtube/YoutubeInfoScraper.java

```diff
@@ -92,18 +93,25 @@ public YoutubeInfoScraper() throws IOException {
         PlaylistItemListResponse response = youTubeClient.playlistItems().list("snippet").setMaxResults(50L)
                 .setPlaylistId(uploadId).execute();
         // getItems() return null when no items match the criteria (uploadId).
-        if (response.getItems() == null || response.getItems().isEmpty()) {
+        if (response.getItems() == null) {
             return Optional.empty();
-        }
+         }
+         if (response.getItems().isEmpty()) {
+            return Optional.empty();
+         }
         return Optional.of(response.getItems());
     }
 
     private Optional<String> getYoutubeChannelResponse(ChannelListResponse response) {
         // getItems() return null when no items match the criteria (channelId).
-        if (response.getItems() == null || response.getItems().isEmpty()) {
+        if (response.getItems() == null) {
+            return Optional.empty();
+        }
+        if (response.getItems().isEmpty()) {
             return Optional.empty();
         }
-        // Since we are only requesting one channel-id, we only get one item back.
+        checkState(response.getItems().size() == 1, "We should only be requesting a single channelId but got "
```

Comment:
Does vscode have a refactorer to turn this into String.format()?  (intellij does)